apps/api/src/app/integrations/e2e/create-integration.e2e.ts

```diff
@@ -23,13 +23,13 @@ describe('Create Integration - /integration (POST)', function () {
   it('should get the email integration successfully', async function () {
     const integrations = (await session.testAgent.get(`/v1/integrations`)).body.data;
 
-    const emailIntegrations = integrations.filter(
+    const emailIntegrations: any[] = integrations.filter(
       (searchIntegration) => searchIntegration.channel === ChannelTypeEnum.EMAIL
     );
 
-    expect(emailIntegrations.length).to.eql(2);
+    expect(emailIntegrations.length).to.eql(3);
 
-    for (const emailIntegration of emailIntegrations) {
+    for (const emailIntegration of emailIntegrations.splice(1, 1)) {
```

Comment:
I am sorry, can't understand what you try to do here. 😞

Reply:
We do it because the last one will be Novu free provider and we do not like to check it here.

Reply:
I changed it to make more sense :)